<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCharView</class>
 <widget class="QWidget" name="CreateCharView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>324</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>469</width>
    <height>324</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>469</width>
    <height>324</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="charBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_validate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>254</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Validate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>194</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_spritePerso1">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>30</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_spritePerso2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
